coding/json.testfile

```diff
@@ -1,3 +1,3 @@
-/multicodec
+/mdagv1
```

Comment:
i think this should have a `/multicodec` first still. im not 100% sure, i need to revisit.